src/components/Drawer/Drawer.docs.mdx

```diff
@@ -0,0 +1,14 @@
+import { Meta, Story } from '@storybook/addon-docs/blocks';
+import { Figma } from 'storybook-addon-designs/blocks';
+
+<Meta title="components/Drawer/Docs" />
+
+# Drawer documentation
+
+>**[Confluence page](https://zitenote.atlassian.net/wiki/spaces/DSA/pages/3183379286/Drawer)** about pattern usage, rationale, planned changes, and any current discrepancies between Storybook and Figma.
+
+> **Visual documentation**: please open in Figma for full context, these are only the most prominent examples of the pattern.
+
+<Figma url="https://www.figma.com/file/vnzYfrwIISRYkr2iYE5TcD/branch/DsLAjWuI0ZmY6lsQ65KoMc/DS-Atoms?node-id=15515%3A22020"/>
+
+To implement the drawer and drift in the page please check the [Drawer implementation](https://zitenote.atlassian.net/wiki/spaces/DSA/pages/3183379286/Drawer#Platform-implementation)
```

Comment:
This can be problematic as the storybook and this repository are public. We probably shouldn't reveal links to our internal systems

Reply:
removed